hdr_html_canvas_element.md

```diff
@@ -283,6 +280,57 @@ the Content Light Level Information chunk can provide more accurate information
 than the maximum luminance parameter of the Mastering Display Color Volume
 chunk.
 
+As specified below, the platform does not generally apply color volume mapping
+if the color volume of the image is smaller than that of the display.
+
+## Add display color volume information to `Screen` interface of the CSS Object Model
+
+Add a new `displayColorVolume` attribute to the `Screen` interface:
+
+```idl
+partial interface Sceeen {
+  optional ColorVolumeInfo displayColorVolume;
+}
+```
+
+If present, the `displayColorVolume` attribute specifies the set of colors that
+the screen of the output device can reproduce without significant color volume
+mapping.
```

Comment:
Yes, I think `displayColorVolume` should be used only for high-performance applications that require precise and accurate knowledge of the display capabilities; for everything else, why not use `dynamic-range` and `color-gamut` media queries?

Reply:
... there is also a much larger issue, which was brought up in the past: knowing the gamut and luminance range of a display is not really useful without also knowing the viewing environment (ambient light level, color, etc.). So it begs the question of how useful `displayColorVolume` is without also a means of parameterizing the viewing environment.

Specifying a complete output color system, e.g. Rec. ITU-T BT.2100 PQ, addresses this since such color system include a viewing environment. Note that the signaled viewing environment (dark surround in the case of  Rec. ITU-T BT.2100 PQ) might not match the current viewing environment (broad daylight), leaving the task to adapting the image to the platform/display.

Do we need to go into this level of detail for out initial pass at an HDR Canvas API?

Reply:
Good question.

What if you just took a complete color system as a whole, like BT.2100 PQ and HLG? Leave room to have a custom color system as well, if it turns out that parameterising everything and targeting actual light becomes a use case.

That's what HDMI and DisplayPort essentially do.

Reply:
As Pierre says, the viewing environment is built in to the signal for PQ and needs to be adapted, for HLG the equations for adaptation are given within the ITU document.

The main use case for this metadata is for when the user/operating system adjusts the diffuse white level.  You can't just scale the image as that may clip the highlights and will change the perception of the shadows and mid tones.  For example, if you have a 1000 nit monitor, HLG 75% signal level would normally appear at 203 nits.  If you increase the diffuse white to 350 and expect the HDR image or video to follow, you would need either a brighter display, or you need to tonemap the highlights to prevent clipping.  (I'm working on some example code for this)

Reply:
Right, but where can we find this `displayColorVolume` / `ScreenColorInfo` metadata?

A browser could expect to get it from the OS, but where would the OS get it from?